SDX-Simple/SAFE_SDX/src/main/java/exoplex/common/utils/PathUtil.java

```diff
@@ -12,6 +12,8 @@ public static String joinFilePath(String dir, String file){
       dirPath = Paths.get(System.getProperty("user.dir"),dir.replaceFirst("\\./",""));
     }else if(!dir.startsWith("/")){
       dirPath = Paths.get(System.getProperty("user.dir"),dir);
+    }else{
```

Comment:
We need to add a space in front of "else" and another after it.

Reply:
I don't know how to replay to your comments below.  I will need to rebase those commits before I push them to the repo in the future. But I cannot merge all changes in one commit, as that will makes it difficult for future changes like revert. Anyway, there would be multiple pull requests with a few commits in each pull request or a single pull requests with a lot of commits for those changes

Reply:
You're right. Merging all changes in one commit isn't an option. I think what we need is to make pull requests by topic: each pull request has a central topic (e.g., fixing a bug or adding a feature or part of a feature) and contains only commits related to its topic. This can make the review process more focused as there is a central thing that one can grasp.  This is also a way to improve the organization of the repo history because commits are well grouped at the time they're merged into a branch.  Of course, if all the commits we have now are about the same topic, we can put them in one pull request.

Reply:
Thanks. This makes sense. I will merge those commits in a single pull request when this part is ready,